convex/dictionary.ts

```
@@ -0,0 +1,580 @@
+    includePublic: v.optional(v.boolean()),
+  },
+  handler: async (ctx, args) => {
+    const user = await ctx.db.get(args.userId);
+    if (!user) {
+      throw new Error("User not found");
+    }
+
+    const limit = args.limit ?? 20;
+    const includePublic = args.includePublic ?? false;
+
+    let query = ctx.db
```

ESLint

[error] 'query' is never reassigned. Use 'const' instead.

(prefer-const)